### PR TITLE
[clusteragent/clusterchecks/dispatcher_rebalance] Fix busyness avg calculation

### PR DESCRIPTION
**Pull request by davidor**
### What does this PR do?

Fixes an error in the calculations performed by the algorithm that rebalances cluster checks.

The average busyness score was not calculated correctly.

This should help in distributing the cluster checks more evenly across check runners, but notice that the rebalance algorithm has other known limitations, so it might not always help.

### Describe how to test/QA your changes

Probably the best is to check that cluster checks are more evenly distributed than before in clusters with several checks and the rebalancing enabled ("cluster_checks.advanced_dispatching_enabled" set to true). But notice that as mentioned above, it might not always be the case.

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature. 
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

**Review by AliDatadog — APPROVED** [review]

File: releasenotes-dca/notes/fix-avg-busyness-clusterchecks-dispatcher-a5f0f5bdca3919dd.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Fixed an error in the calculations performed by the algorithm that rebalances cluster checks. Cluster checks should now be more evenly distributed now when advanced dispatching is enabled (``cluster_checks.advanced_dispatching_enabled`` is set to true).
```
Comment (rtrieu) [inline releasenotes-dca/notes/fix-avg-busyness-clusterchecks-dispatcher-a5f0f5bdca3919dd.yaml:11]:
```suggestion
    Fixed an error in the calculations performed by the algorithm that rebalances cluster checks. Cluster checks are now more evenly distributed when advanced dispatching is enabled (``cluster_checks.advanced_dispatching_enabled`` is set to true).
```

**Review by rtrieu — CHANGES_REQUESTED** [review]
Minor suggestion for clarity, but otherwise looks good!

File: releasenotes-dca/notes/fix-avg-busyness-clusterchecks-dispatcher-a5f0f5bdca3919dd.yaml (hunk 1 above)
Comment (davidor) [reply releasenotes-dca/notes/fix-avg-busyness-clusterchecks-dispatcher-a5f0f5bdca3919dd.yaml]: Changed :+1:

**Review by rtrieu — APPROVED** [review]
Looks great!

**Pull request merged**